### ISSUE TITLE
LoadResPicture parameter restype does not match with VB6

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#240]
**Describe the bug**
The restype parameter is in tB "ByVal" and "As Long". But it should be "ByRef" and "As Integer".

**Screenshots**
![image](https://user-images.githubusercontent.com/33756666/123457938-bd4afc80-d5e4-11eb-81b2-bf439392f172.png)

![image](https://user-images.githubusercontent.com/33756666/123458013-d358bd00-d5e4-11eb-9a7c-c000f973233d.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.10.2773]

**Additional context**
I think that the return value "As Variant" is by intent ? (Thus no dependency on stdole lib)

**Comment by Kr00l** [comment]
addition:
Maybe it would be better to return "As Object" so late-bound calls can work ?

Example below code works in VB6 but not in tB.
```
Sub Main()
MsgBox LoadResPicture(101, vbResIcon).Handle
End Sub
```

The workaround works both in tB and VB6
```
Dim Pic As IPictureDisp
Set Pic = LoadResPicture(101, vbResIcon)
MsgBox Pic.Handle
```

**Comment by WaynePhillipsEA** [comment]
Good catch, admittedly I only did these functions quickly for now until I move them over to the proper Globals class, but I'll get them corrected, thanks.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.2780.  Thanks for reporting.

The return type of LoadResPicture is now Object.  I will have it corrected up to IPictureDisp when stdole is referenced, in the next release.

**Comment by WaynePhillipsEA** [comment]
Fixed in v0.10.2781.  Now, if stdole is referenced, then the return type of LoadResPicture is IPictureDisp [Picture] to give better intellisense, and fully match VB6.
